biolinkml/utils/schemaloader.py

```diff
@@ -195,7 +195,7 @@ def resolve(self) -> SchemaDefinition:
         for slot in self.schema.slots.values():
             # Propagate domain to containing class
             if slot.domain and slot.domain in self.schema.classes:
-                if slot.name not in self.schema.classes[slot.domain].slots:
+                if slot.name not in self.schema.classes[slot.domain].slots and not slot.owner:
                     slot.owner = slot.name
                     self.schema.classes[slot.domain].slots.append(slot.name)
```

Comment:
I'm still suspicious of lines like this

Reply:
this line: `self.schema.classes[slot.domain].slots.append(slot.name)`

Reply:
Why?

Reply:
it looks like domains are inducing classes to have slots?